pandas/tests/groupby/test_apply.py

```diff
@@ -752,3 +752,26 @@ def most_common_values(df):
         ["17661101"], index=pd.DatetimeIndex(["2015-02-24"], name="day"), name="userId"
     )
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize("test_series", [True, False])
```

Comment:
this test needs to be close to other tests like this, you'll have to look (likey in this module though)

Reply:
Yea I understand. In the previous commit I didn't think any existing ones are similar to this. Nothing tests unique indices.

Anyway I moved this next to `test_apply_without_copy` which is somewhat similar to this in that they both are related to the execution path where ``mutated`` is ``False`` when combining results.